### ISSUE TITLE
JavaScript not working

### ISSUE THREAD

**Issue by dominikgeimer** [cesaramirez/laravel-tabler#25]
Hi,

I am using your packages with Laravel 5.7 and followed the instructions. Everythings looks fine, but the javascript part is not working. For example the Dropdowns in the Navigation.

Any Idea?

**Comment by cesaramirez** [comment]
Hello
Thank you for notifying, I will review to what this error is due.
regards

**Comment by EduardoRT** [comment]
Same thing happening to me, every click on the site triggers an error.

```
Uncaught DOMException: Failed to execute 'querySelector' on 'Document': 'javascript:void(0)' is not a valid selector.
    at Object.getSelectorFromElement (https://localhost:3000/js/app.js?id=1881ba3a583e42d488f7:1364:35)
    at Function._getParentFromElement (https://localhost:3000/js/app.js?id=1881ba3a583e42d488f7:3120:27)
    at HTMLDocument._clearMenus (https://localhost:3000/js/app.js?id=1881ba3a583e42d488f7:3074:31)
    at HTMLDocument.dispatch (https://localhost:3000/js/app.js?id=1881ba3a583e42d488f7:10986:27)
    at HTMLDocument.elemData.handle (https://localhost:3000/js/app.js?id=1881ba3a583e42d488f7:10794:28)```

**Comment by cesaramirez** [comment]
Fix #26
